components/sections/repo-questions.js

```diff
@@ -87,10 +113,56 @@ const RepoQuestions = ({ data }) => {
       {data.repo_question.map((q, i) => {
         if (i + 1 === questionToShow) {
           return (
-            <div className="repo-questions" key={q.question}>
-              <Markdown className="repo-questions">{q.question}</Markdown>
-              <br></br>
-              {showOptions && (
+            <div key={q.question}>
+              <div>
+                {questionToShow}. <Markdown inline>{q.question}</Markdown>
+              </div>
+              <br />
+
+              {q.options.length > 2 && (
```

Comment:
This works based on the questions that we have now, but it seems possible that they could want a radio form with more than two options or a checkbox form with 2 options. So it may be worth the extra effort to make this more flexible in the future. On the vlmd quiz, I added an [extra key to the questions](https://github.com/heal-data-stewards/website/blob/e6d4351e7e203f88dc40e49b9393bb46762548e0/components/sections/vlmd/questions.json) to define what type of form the question should be ("single-choice", "multi-choice", etc)